cmd/server-mux.go

```diff
@@ -420,21 +451,110 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 		MinVersion: tls.VersionTLS12,
 	} // Always instantiate.
 
-	if tlsEnabled {
-		// Configure TLS in the server
-		if config.NextProtos == nil {
-			config.NextProtos = []string{"http/1.1", "h2"}
+	// Configure TLS in the server
+	if config.NextProtos == nil {
+		config.NextProtos = []string{"http/1.1", "h2"}
+	}
+	config.Certificates = make([]tls.Certificate, 1)
+
+	config.Certificates[0], tlsErr = tls.X509KeyPair(m.certPEM, m.keyPEM)
+	if tlsErr != nil {
+		return nil, false, false, tlsErr
+	}
+
+	// Fetch for OCSP response
+	ocspBytes, _, ocspErr := ocsp.GetOCSPForCert(m.certPEM)
```

Comment:
> As ocsp.GetOCSPForCert() returns ocsp.Response which contains status of the certificate Status is one of {Good, Revoked, Unknown}. Don't we need to handle the response properly?

We have to ignore 'Good' and 'Unknown' but I am not really sure for 'Revoked', if the server receives this latter, it means that its certificate is not valid anymore. Clients are already alerted. I still think that the server should run normally until the admin replaces the certificate.